src/utils/util.js

```diff
@@ -34,6 +34,5 @@ export const handleRegister = (username, email, password, passwordConfirmation)
   username,
   email,
   password,
-  passwordConfirmation,
-},
-{ withCredentials: true });
+  password_confirmation: passwordConfirmation,
```

Comment:
This parameter is the one being permitted [here](https://github.com/verissimohenry/verissimohenry-Find-Your-House-backend/blob/e9502063a5ae3d4d0beae32c60013b4d135e4f20/app/controllers/registrations_controller.rb#L16).

Reply:
ok